CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ All notable changes to Kafka extension will be documented in this file.
 - Improved the "New cluster" and "New topic" wizards: now include validation and a back button. See [#21](https://github.com/jlandersen/vscode-kafka/issues/21).
 - Newly created topic or cluster is automatically selected in the Kafka Explorer. See [#61](https://github.com/jlandersen/vscode-kafka/issues/61).
 - Internal topics are now hidden by default. See [#29](https://github.com/jlandersen/vscode-kafka/issues/29) and [#74](https://github.com/jlandersen/vscode-kafka/pull/74).
+- Elements should be sorted alphabetically in the explorer. See [#63](https://github.com/jlandersen/vscode-kafka/issues/63).
```

Comment:
Elements are now sorted alphabetically in the Kafka explorer

Reply:
fixed